output/cloud/v1/output.go

```diff
@@ -472,4 +514,42 @@ func (out *Output) pushMetrics() {
 	}).Debug("Pushing metrics to cloud finished")
 }
 
+func (out *Output) runFlushRequestMetadatas() {
+	t := time.NewTicker(out.config.TracesPushInterval.TimeDuration())
+
+	for i := int64(0); i < out.config.TracesPushConcurrency.Int64; i++ {
+		out.outputDone.Add(1)
+		go func() {
+			defer out.outputDone.Done()
+			defer t.Stop()
+
+			for {
+				select {
+				case <-out.stopSendingMetrics:
+					return
+				default:
+				}
+				select {
+				case <-out.stopOutput:
+					out.flushRequestMetadatas()
+					return
+				case <-t.C:
+					out.flushRequestMetadatas()
+				}
+			}
+		}()
+	}
+}
+
+func (out *Output) flushRequestMetadatas() {
+	start := time.Now()
+
+	err := out.requestMetadatasFlusher.Flush()
+	if err != nil {
+		out.logger.WithError(err).WithField("t", time.Since(start)).Error("Failed to push trace samples to the cloud")
+	}
+
+	out.logger.WithField("t", time.Since(start)).Debug("Successfully flushed buffered trace samples to the cloud")
```

Comment:
In case of an error, this log entry still will be written. This applies to the v2 output too.

Reply:
Great catch, fixed in [25ec9fa](https://github.com/grafana/k6/pull/3202/commits/25ec9fa9489f1f319044bce7aeac8a94a5776ff0).